ratis-datastream/src/main/java/org/apache/ratis/datastream/objects/DataStreamReply.java

```diff
@@ -0,0 +1,7 @@
+package org.apache.ratis.datastream.objects;
+
+public class DataStreamReply {
```

Comment:
Change both DataStreamRequest and DataStreamReply to interfaces so that we will have methods like getStreamId() but no fields.